helm-frontend/src/components/InstanceData.tsx

```diff
@@ -25,7 +26,11 @@ export default function InstanceData({
         {`Instance id: ${instance.id} [split:  ${instance.split}]`}
       </h3>
       <h3>Input</h3>
-      {instance.input.text.includes(`<br><img src="data:image;base64`) ? (
+      {instance.input.multimedia_content !== undefined ? (
+        <MultimediaObjectDisplay
+          multimediaObject={instance.input.multimedia_content}
+        />
+      ) : instance.input.text.includes(`<br><img src="data:image;base64`) ? (
         <div dangerouslySetInnerHTML={{ __html: instance.input.text }} />
```

Comment:
We can remove this support after the other PR is merged right?

Reply:
Correct!